src/server/user-controller.js

```diff
@@ -1,15 +1,21 @@
 import express from 'express';
```

Comment:
Bonus commit here:

- moved the methods from the `user-request-handler` in here to follow the new controller pattern (since I had to touch this file anyway)

Reply:
:+1: